suite-native/receive/src/hooks/useAccountReceiveAddress.tsx

```diff
@@ -104,13 +119,20 @@ export const useAccountReceiveAddress = (accountKey: AccountKey) => {
                 showAlert({
                     title: response.payload.payload.code,
                     description: response.payload.payload.error,
-                    icon: 'warningCircle',
-                    pictogramVariant: 'red',
-                    primaryButtonTitle: 'Cancel',
+                    primaryButtonVariant: 'redBold',
+                    primaryButtonTitle: <Translation id="generic.buttons.tryAgain" />,
                     onPressPrimaryButton: () => {
-                        TrezorConnect.cancel();
-                        navigation.goBack();
-                        setIsUnverifiedAddressRevealed(false);
+                        handleCancel();
+                        // Handle show address calls verifyAddressOnDevice and here we want to trigger it again so there is cyclic dependency
+                        // between these calls.
+                        // eslint-disable-next-line @typescript-eslint/no-use-before-define
```

Comment:
why this eslint-disable?

Reply:
It's explained in the comment. Is it still unclear?